buzzmoon-frontend/buzzmoon-vite/src/Components/BackendActor/backend-actor.js

```diff
@@ -250,4 +250,55 @@ export default class BackendActor {
     return response;
   }
 
+  static resultsToHeadToHeadTable(results) {
+    let response = []
+    results.map((r) => {
+      let w = 0;
+      let l = 0;
+      let t = 0;
+      let outcomes = [];
+      results.filter((e) => (e.name !== r.name)).map((o) => {
```

Comment:
looks like o stands for opponent I suppose? what is e? i understand you might want to try to keep it short, but might be easier to follow if you at least define these as opponent and rest of vars can stay the same (i.e oscore) 

also ocel? maybe do ocelery?